_layouts/default.html

```diff
@@ -9,7 +9,7 @@
       {% include analytics.html %}
     {% endif %}
   </head>
-  <body class="{{ layout.class }} {{ page.class }}">
+  <body class="{{ layout.class }} {{ page.class }} {{ site.theme_variant }}">
```

Comment:
Yeah, I actually don't know how useful this will be since we will be including our own stylesheet after all of the USWDS so specificity shouldn't be an issue, but it doesn't hurt having it included either.

Reply:
Yeah I was actually thinking of removing this

Reply:
I like it. Would give me an easy hook if I wanted to override anything. But then, I like meaningful body classes.

Reply:
I think we can cut this out now that we are moving away from named themes.